dev/io.openliberty.org.apache.taglibs.standard/bnd.bnd

```diff
@@ -31,7 +31,10 @@ Export-Package: \
    org.apache.taglibs*;version=2.0
 
 -privatepackage: \
-  com.oracle.wls.shaded.*
+  !com.oracle.wls.shaded.org.apache.xalan.xsltc.*,\
+  com.oracle.wls.shaded.org.apache.xalan.*, \
+  com.oracle.wls.shaded.org.apache.xml.*, \
+  com.oracle.wls.shaded.org.apache.xpath.*
```

Comment:
I'm guessing these 3 packages make up the remainder of what's in `com.oracle.wls.shaded.` outside of `com.oracle.wls.shaded.org.apache.xalan.xsltc.`?

Reply:
Thanks for the review @mbroz2! Yes that makes up the packages we have a dependency on. A bit more context on this is that the XSLTC package is the affected package which also requires dependencies bundled in Xalan like BCEL (which seemed to be where the origin of the vulnerability was found). Since we remove that XSLTC package, then we no longer need the extra dependencies therefore we only pack the required xalan, xml, and xpath packages. This is mimicking a build instruction of Xalan called the interpretive Xalan version as seen [here](https://github.com/apache/xalan-java/blob/master/build.xml#L654-L675) which as explained by their help message as `Jar up everything in Xalan interpretive (without XSLTC)` so it is a version of Xalan not packaging XSLTC which is just what we needed. To not upload and maintain our own version of Xalan, we decided to munge it in this way.

Reply:
Will this have a negative effect on performance?  It seems like it is now using interpretive instead of compiling.  Was the xsltc code being called even or was it just there?

Reply:
It shouldn't have any effect, the xsltc code wasn't being used here. For compiling and transforming with stylesheets, the provided JDK XML transformations are used.